lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -31,7 +35,17 @@ export class BrowserStorage implements IWindowStorage<string> {
     }
 
     setItem(key: string, value: string): void {
-        this.windowStorage.setItem(key, value);
+        try{
+            this.windowStorage.setItem(key, value);
+        }
+        catch(e){
+            if(e.code === BrowserAuthErrorMessage.chromiumStorageException.code || e.code === BrowserAuthErrorMessage.firefoxStorageException.code)
```

Comment:
What about other browsers? Is there no standard error?

Reply:
There are browser specific error codes , and error most common being QuotaExceededError and QUOTA_EXCEEDED_ERR. Alternatively , to avoid using this if check, any exception thrown during setItem() can be regarded as a storage error since there is already a method call to validate the existence of storage in the constructor. 
I couldnt find a standard error name or code that is applicable across all browser environments.